PracticeGit/TicTacToe.c

```diff
@@ -0,0 +1,43 @@
+// ------------------------------------------------------------------------------------------------
+// Training ~ A training program for new joiners at Metamation, Batch - July 2024.
+// Copyright (c) Metamation India.
+// ------------------------------------------------------------------------------------------------
+// TicTacToe.c - Program to implement a two-player Tic-Tac-Toe game.
+// ------------------------------------------------------------------------------------------------
+#include "TicTacToe.h"
+
+char board[3][3];
+int currentPlayer = 1;
+char playerSymbols[] = { 'X', 'O' };
+
+void initializeBoard () {
+   for (int i = 0; i < 3; i++)
+      for (int j = 0; j < 3; j++) board[i][j] = ' ';
+}
+
+int checkWinner () {
+   for (int i = 0; i < 3; i++) {
+      if (board[i][0] == board[i][1] && board[i][1] == board[i][2] && board[i][0] != ' ') return 1; // Check rows
+      if (board[0][i] == board[1][i] && board[1][i] == board[2][i] && board[0][i] != ' ') return 1; // Check coloumns
```

Comment:
The last check board[i][0] != ' ' must be performed first before comparing other rows and columns, so that as the first condition fails it will skip the upcoming conditions from checking. similarly for diagonals also. Check the spellings for the comments as well.